module/VuFind/src/VuFind/XSLT/Import/VuFind.php

```diff
@@ -64,13 +64,13 @@ public static function setServiceLocator($serviceLocator)
     }
 
     /**
-     * Get the change tracker table object.
+     * Get the change tracker service object.
      *
-     * @return \VuFind\Db\Table\ChangeTracker
+     * @return \VuFind\Db\Service\ChangeTrackerService
      */
     public static function getChangeTracker()
     {
-        return static::$serviceLocator->get(\VuFind\Db\Table\PluginManager::class)
+        return static::$serviceLocator->get(\VuFind\Db\Service\PluginManager::class)
```

Comment:
Should we change 'ChangeTracker' below to the fully-qualified class name? The alias should work too, but I think we've been using the FQCN in most situations, so it might be more consistent to do the same here.